example/babel.config.js

```diff
@@ -1,5 +1,5 @@
-const path = require('path');
-const pak = require('../package.json');
+// const path = require('path');
```

Comment:
why are these commented out?

Reply:
The project was misconfigured initially and doing some custom funky path/alias stuff that wasn't playing nicely, so I updated it to match a more standard package usage. I can just delete these though, no reason to leave them commented out.